src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,151 @@
 package core.basesyntax;
 
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) throws IOException {
+        Scanner scanner = new Scanner(System.in);
+        String command = scanner.next();
+        byte exit = 1;
+        while (exit == 1) {
```

Comment:
тут краще зробити так:
```suggestion
        while (true) {
```